en/docs/release-notes/version-4.4.0.md

```diff
@@ -0,0 +1,17 @@
+# Ballerina VS Code extension version 4.4.0
+
+We are happy to announce the Ballerina VS Code extension 4.4.0 release with a few exciting new features and improvements. Following are the highlights of this release.
+
+If you are new to Ballerina, you can download the [installers](https://ballerina.io/downloads/) to install it. You can install the Ballerina VS Code extension from the [VS Code marketplace](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina).
+
+## New features
+
+- New release now supports opening ballerina gist files and repositories using the vscode URL command.
+  - To open a gist file:
+      - vscode://wso2.ballerina/open-file?gist=74cea880fefcb463d26a0c46f38fce39&file=hello_world.bal
+      - vscode://wso2.ballerina/open-repo?repoUrl=https://github.com/wso2/choreo-sample-apps
+
+
+## Improvements
+
+- The GraphQL designer, Entity-relationship diagram, Type diagram, and Composition diagram, now provide support for users to navigate to a specific node based on their selection in the field type.
```

Comment:
Entity-Relationship diagram, and Type Diagram.

We do not support this for composition diagram yet.